pkg/operator/status.go

```diff
@@ -125,10 +125,6 @@ func (optr *Operator) syncProgressingStatus() error {
 }
 
 func (optr *Operator) updateStatus(co *configv1.ClusterOperator, status configv1.ClusterOperatorStatusCondition) error {
-	existingCondition := cov1helpers.FindStatusCondition(co.Status.Conditions, status.Type)
-	if existingCondition.Status != status.Status {
-		status.LastTransitionTime = metav1.Now()
-	}
```

Comment:
any reason to have dropped this?

Reply:
It was causing a nil-pointer dereference panic when the existing condition wasn't found for the new type, but the logic is already handled in the call to `cov1helpers.SetStatusCondition` anyways so I just removed it.

https://github.com/openshift/library-go/blob/3679c350479175dceb60cbdafe68f0d92280952f/pkg/config/clusteroperator/v1helpers/status.go#L31

Reply:
👍👍